ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/subscription/request/SubscriptionRequestMapperTest.java

```diff
@@ -54,83 +55,88 @@ public void before() {
 
   @Test
   public void mapRequestToUnsubscribeRequest() {
-    final JsonRpcRequest jsonRpcRequest =
+    final JsonRpcRequestBody jsonRpcRequest =
```

Comment:
nit: these request variable names should be jsonRpcRequestBody instead

Reply:
done